README.md

```diff
@@ -1,3 +1,5 @@
+# THIS IS WORK IN PROGRESS for version 1.0 of
```

Comment:
This is not needed anymore, as the repo is not being cloned.

Reply:
Let's clean that up together with the Paradox doc PR.